File: app/src/main/java/com/hpinc/voter/VoteActivity.java
package com.hpinc.voter;

import android.os.Bundle;
import android.app.Activity;
import android.content.Intent;
import android.util.Log;
import android.view.View;
import android.view.View.OnClickListener;
import android.widget.Button;
import android.widget.RadioButton;
import android.widget.RadioGroup;
import android.widget.RadioGroup.OnCheckedChangeListener;
import android.widget.Toast;

import com.example.feedback.R;


public class VoteActivity extends Activity {
	
	private RadioGroup radioGroup;
	private RadioButton candidate1,candidate2,nato;
	private Button button;

    DatabaseHelper db = new DatabaseHelper(VoteActivity.this);
	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_vote);
		
		
		radioGroup = (RadioGroup) findViewById(R.id.radioGroup1);
		
	
		
		radioGroup.setOnCheckedChangeListener(new OnCheckedChangeListener() {

			@Override
			public void onCheckedChanged(RadioGroup group, int checkedId) {
				// find which radio button is selected
				if(checkedId == R.id.radio0) {
					Toast.makeText(getApplicationContext(), "choice: ASSA", Toast.LENGTH_SHORT).show();
				} else if(checkedId == R.id.radio1){
					Toast.makeText(getApplicationContext(), "choice: PHPRY", Toast.LENGTH_SHORT).show();
				}
                else if(checkedId == R.id.radioButton5)
                {
                    Toast.makeText(getApplicationContext(), "choice: NOTA", Toast.LENGTH_SHORT).show();
                }
				else
				{
					Toast.makeText(getApplicationContext(), "choose any one option", 
							Toast.LENGTH_SHORT).show();	
				}
			}
		});
	
		
		candidate1 = (RadioButton) findViewById(R.id.radio0);
		candidate2 = (RadioButton) findViewById(R.id.radio1);
	    nato= (RadioButton) findViewById(R.id.radioButton5);
	   
	    button = (Button)findViewById(R.id.button1);
	    button.setOnClickListener(new OnClickListener() {
			
			@Override
			public void onClick(View v) {
				int selectedId = radioGroup.getCheckedRadioButtonId();
				
				// find which radioButton is checked by id
				if(selectedId == candidate1.getId()) {
					Toast.makeText(getApplicationContext(),"You chose 'ASSA'",Toast.LENGTH_SHORT).show();
					RegisterActivity.count[db.getyourdata2(LoginActivity.user, LoginActivity.pass)]=1;
					
				}
				else if(selectedId == candidate2.getId()){
					Toast.makeText(getApplicationContext(),"You chose 'PHPRY'",Toast.LENGTH_SHORT).show();
                    RegisterActivity.count[db.getyourdata2(LoginActivity.user, LoginActivity.pass)]=1;
					}
                else if(selectedId == nato.getId()){
                    Toast.makeText(getApplicationContext(),"You chose 'NOTA'",Toast.LENGTH_SHORT).show();
                    RegisterActivity.count[db.getyourdata2(LoginActivity.user, LoginActivity.pass)]=1;
                }
				else {
					Toast.makeText(getApplicationContext(),"you did not choose any option",Toast.LENGTH_SHORT).show();	
				}

				Intent i =new Intent(getApplicationContext(),ResultActivity.class);
				
				startActivity(i);
			}
			});
	    
	}public void onBackPressed() {
        Log.d("CDA", "onBackPressed Called");
        Intent setIntent = new Intent(Intent.ACTION_MAIN);
        setIntent.addCategory(Intent.CATEGORY_HOME);
        setIntent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
        startActivity(setIntent);
    }

}

File: app/src/main/java/com/hpinc/voter/VoteActivity.kt
package com.hpinc.voter

import android.os.Bundle
import android.app.Activity
import android.content.Intent
import android.database.sqlite.SQLiteDatabase
import android.util.Log
import android.view.View
import android.widget.Button
import android.widget.RadioButton
import android.widget.RadioGroup
import android.widget.Toast

import com.example.feedback.R


class VoteActivity : Activity() {

    private var radioGroup: RadioGroup? = null
    private var candidate1: RadioButton? = null
    private var candidate2: RadioButton? = null
    private var nato: RadioButton? = null
    private var button: Button? = null

    private var db = DatabaseHelper(this@VoteActivity)
    private lateinit var sb: SQLiteDatabase
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_vote)
        sb = db.readableDatabase

        radioGroup = findViewById<View>(R.id.radioGroup1) as RadioGroup



        radioGroup!!.setOnCheckedChangeListener { group, checkedId ->
            // find which radio button is selected
            if (checkedId == R.id.radio0) {
                Toast.makeText(applicationContext, "choice: ASSA", Toast.LENGTH_SHORT).show()
            } else if (checkedId == R.id.radio1) {
                Toast.makeText(applicationContext, "choice: PHPRY", Toast.LENGTH_SHORT).show()
            } else if (checkedId == R.id.radioButton5) {
                Toast.makeText(applicationContext, "choice: NOTA", Toast.LENGTH_SHORT).show()
            } else {
                Toast.makeText(applicationContext, "choose any one option",
                        Toast.LENGTH_SHORT).show()
            }
        }


        candidate1 = findViewById<View>(R.id.radio0) as RadioButton
        candidate2 = findViewById<View>(R.id.radio1) as RadioButton
        nato = findViewById<View>(R.id.radioButton5) as RadioButton

        button = findViewById<View>(R.id.button1) as Button
        button!!.setOnClickListener {
            val selectedId = radioGroup!!.checkedRadioButtonId
            // find which radioButton is checked by id
            when (selectedId) {
                candidate1!!.id -> {
                    Toast.makeText(applicationContext, "You chose 'ASSA'", Toast.LENGTH_SHORT).show()
                    var query = "UPDATE LOGGER SET voted='true' WHERE first_name='"+ LoginActivity.user +"' and password='"+LoginActivity.pass+"'"
                    sb.execSQL(query)
                    query = "UPDATE LOGGER SET castedTo='ASSA' WHERE first_name='"+ LoginActivity.user +"' and password='"+LoginActivity.pass+"'"
                    sb.execSQL(query)
                }
                candidate2!!.id -> {
                    Toast.makeText(applicationContext, "You chose 'PHPRY'", Toast.LENGTH_SHORT).show()
                    var query = "UPDATE LOGGER SET voted='true' WHERE first_name='"+ LoginActivity.user +"' and password='"+LoginActivity.pass+"'"
                    sb.execSQL(query)
                    query = "UPDATE LOGGER SET castedTo='PHPRY' WHERE first_name='"+ LoginActivity.user +"' and password='"+LoginActivity.pass+"'"
                    sb.execSQL(query)
                }
                nato!!.id -> {
                    Toast.makeText(applicationContext, "You chose 'NOTA'", Toast.LENGTH_SHORT).show()
                    var query = "UPDATE LOGGER SET voted='true' WHERE first_name='"+ LoginActivity.user +"' and password='"+LoginActivity.pass+"'"
                    sb.execSQL(query)
                    query = "UPDATE LOGGER SET castedTo='NOTA' WHERE first_name='"+ LoginActivity.user +"' and password='"+LoginActivity.pass+"'"
                    sb.execSQL(query)
                }
                else -> Toast.makeText(applicationContext, "you did not choose any option", Toast.LENGTH_SHORT).show()
            }

            val i = Intent(applicationContext, ResultActivity::class.java)

            startActivity(i)
        }

    }

    override fun onBackPressed() {
        Log.d("CDA", "onBackPressed Called")
        val setIntent = Intent(Intent.ACTION_MAIN)
        setIntent.addCategory(Intent.CATEGORY_HOME)
        setIntent.flags = Intent.FLAG_ACTIVITY_NEW_TASK
        startActivity(setIntent)
    }

}


